### ISSUE TITLE
[Bug Report]: REI error

### ISSUE THREAD

**Issue by serways2** [TeamAOF/All-of-Fabric-7#290]
### Modpack Version

2.4.3

### Describe your issue.

started crashing after updating the build to version 2.4.3
writes that this is possible due to the rendering of objects in REI

### Crash Report

<script src="https://gist.github.com/serways2/175e17abdad99d126502b5e0a373de66.js"></script>

### Latest Log

<script src="https://gist.github.com/serways2/c942f9678c2052a8d95241ca772f3da9.js"></script>

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

**Comment by serways2** [comment]
Latest Log
https://gist.github.com/serways2/c942f9678c2052a8d95241ca772f3da9
Crash Report
https://gist.github.com/serways2/72e202cd7725d0989c26d0a1b88dd329

**Comment by perci100** [comment]
https://mclo.gs/6zMZOYB

Similar issue today.

**Comment by sutheman86** [comment]
Faced the same issue, update Variants and Ventures to 1.0.4 would solve this issue

**Comment by perci100** [comment]
> Faced the same issue, update Variants and Ventures to 1.0.4 solved this.

Updating to 1.04 did not fix this for me. I cannot search 'skull' in REI, as both the gelid head and the murk skull insta-crash my REI

**Comment by sutheman86** [comment]
oh my bad, I accidentally installed forge version and it wasn't even loaded. I installed the correct version and same issue as you appears.
However, this issue is not caused by REI, but this mod only. Because I searched 'gelid head' and 'murk skull' in creative and the game freezed.
This mod crashed with fabric 0.15.11, but could work normally with newest version of fabric loader.
here's the issue posted on GitHub: [link](https://github.com/Faboslav/variants-and-ventures/issues/23)

**Comment by perci100** [comment]
> oh my bad, I accidentally installed forge version and it wasn't even loaded. I installed the correct version and same issue as you appears. However, this issue is not caused by REI, but this mod only. Because I searched 'gelid head' and 'murk skull' in creative mode menu and the game freezed. This mod crashed with fabric 0.15.11, but could work normally with newest version of fabric loader. here's the issue posted on GitHub: [link](https://github.com/Faboslav/variants-and-ventures/issues/23)

Yeah, I submitted that report to the mod author, hopefully they'll fix it soon (or remove them like they've mentioned)

**Comment by Faboslav** [comment]
Just to give update here as a author of the mod. I would love to fix this issue, but i cant find the steps to reproduce, i am currently trying fabric loader of `0.15.1`, mod version of `1.0.4` and REI, which is working for me. If any of you can join the Friends&Foes discord and try the dev build and see if its working as the fix, that would be very useful!

**Comment by DJteaCosy** [comment]
I just encountered the same error, and was experiencing it while trying to use EMI as well. Searching for "verdant" in REI will instantly crash the instance. I can supply crash logs if you'd like. 

I've since updated the pack to 2.4.5, will comment again if it crashes

**Comment by Faboslav** [comment]
Wait, both of the reports above are with `1.0.3` version of the mod, can anyone share the log with `1.0.4` version of the mod please?

**Comment by Faboslav** [comment]
This should be fixed with the 1.0.5 release available here on github, CF or Modrinth :).
